.github/workflows/release_python.yml

```diff
@@ -77,7 +77,11 @@ jobs:
       matrix:
         include:
           - target: x86_64
+            wheel_suffix: macosx_11_0_x86_64
+          - target: arm64
+            wheel_suffix: macosx_11_0_arm64
```

Comment:
Build an extra `arm64` wheel -- if that is more specific it can avoid downloading the larger universal binary.